### PR TITLE
Created Site Header by Jakhongirs

### PR DESCRIPTION
**Pull request by jakhongirs**
Done!

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -34,3 +33,98 @@ img {
...
+.nav_link {
+  font-size: 14px;
+  line-height: 16px;
+  letter-spacing: -0.107692px;
+  color: #9597a5;
+  text-decoration: none;
+  padding-bottom: 25px;
+  position: relative;
+}
+/* .nav_link::after {
```
Comment (bdevpro) [inline css/style.css:100]: manashu commentlani olib tashlash kerak

**Review by bdevpro — CHANGES_REQUESTED** [review]
yaxshi or faqat sal muammolar bor

File: css/style.css (hunk 1 above)
Comment (jakhongirs) [reply css/style.css:100]: WIP

File: css/style.css [hunk 2]
```diff
@@ -34,3 +33,90 @@ img {
...
+
+.header {
+  position: fixed;
+  background-color: #fff;
+  top: 0;
+  left: 0;
+  width: 100%;
+  padding-top: 18px;
+  padding-bottom: 18px;
+  z-index: 1000;
```
Comment (bdevpro) [inline css/style.css:59]: `z-index:10000; ` kopaytirish kerak

**Review by bdevpro — CHANGES_REQUESTED** [review]
sal muammolar bor

File: css/style.css (hunk 2 above)
Comment (jakhongirs) [reply css/style.css:59]: Bo'ldi brat hozir qilamiz

**Pull request merged**